src/commenttarget.js

```diff
@@ -0,0 +1,63 @@
+const SEPARATOR = '#';
```

Comment:
Should we use this instead (doesn't require escaping, i.e. `--target="pr#1"` => `--target=pr/1`?

```suggestion
const SEPARATOR = '/';
```

Reply:
Yes, because `echo --target pr#1` => `--target pr` since `#` is used for inline shell comments

Reply:
The change is up for review [here](https://github.com/iterative/cml/pull/1279)